### PR TITLE
Use checksum validation in vendorme for kyverno

### PR DESCRIPTION
**Pull request by trmiller**
Previous iteration used a straight download of the file without any checks.
This will use the sha256 sum of file to validate.  Not perfect, but better.

### CODE REVIEW

File: platform/vendor/vendor.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -euo pipefail
 
-go install github.com/trmiller/vendorme@91071b4ac30f03e42774f72ebf3fd80f57a710ea
+go install github.com/trmiller/vendorme@0b7091a0736be19ca2fd0a3245d997356822ba07
 
 vendorme pull
```
Comment (bradbeck) [inline platform/vendor/vendor.sh:6]:
```suggestion
pushd "$GIT_ROOT/platform/vendor"
vendorme pull
popd
```

File: platform/vendor/vendor.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -euo pipefail
```
Comment (bradbeck) [inline platform/vendor/vendor.sh:3]:
```suggestion
# Define variables.
GIT_ROOT=$(git rev-parse --show-toplevel)

```

File: platform/vendor/vendor_kyverno.sh [hunk 3]
```diff
@@ -1,28 +0,0 @@
...
-KYVERNO_RESOURCE_DIR=${GIT_ROOT}/resources/kyverno/admission-control-policy
-
-### HACK to work around the latest version of Kyverno not having the updates Jim made for us with configmaps. 
-### Once offically released. We will go back to using the versioned release.
-KYVERNO_URL="https://raw.githubusercontent.com/kyverno/kyverno/main/config/install.yaml"
-
-# # Define variables.
-# KYVERNO_TAG_RELEASE=v1.5.1
-# KYVERNO_URL="https://raw.githubusercontent.com/kyverno/kyverno/$KYVERNO_TAG_RELEASE/definitions/release/install.yaml"
-# COSIGN_REPOSITORY=ghcr.io/kyverno/signatures
```
[the remark this answers is not in the document]
Comment (trmiller) [reply platform/vendor/vendor_kyverno.sh:15]: I'm inclined to just delete it given it's part of their [documentation for validating the install](https://kyverno.io/docs/installation/#verifying-kyverno-image-signatures-using-cosign).  Separately going to ask them if they are planning on using rekor.  Open to whatever though.

**Review by bradbeck — APPROVED** [review]

File: platform/vendor/vendor_kyverno.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (pxp928) [reply platform/vendor/vendor_kyverno.sh:15]: So we dont want to validate in our repo with cosign?

[the remark this answers is not in the document]
Comment (trmiller) [reply platform/vendor/vendor_kyverno.sh:15]: Not saying that - was saying 1) we're currently not, those were just old comments and didn't feel the need to keep them around given it's still documented on their page.  2) I'd prefer rekor, though unclear if they're going to do that - will ask them.  3) will add cosign verification back if 2 won't happen, or won't be for a while.  cosign > hash.

**Review by pxp928 — APPROVED** [review]

**Pull request merged**